README.md

```diff
@@ -115,4 +115,3 @@ directory.
 
 [shippable]: https://app.shippable.com/projects/577032be3be4f4faa56adb38
 [shippable img]: https://img.shields.io/shippable/577032be3be4f4faa56adb38/master.svg?label=shippable
-
```

Comment:
Please restore empty line at end of file.
I'm surprised github doesn't show red arrow for it on file view.